### PR TITLE
Update Set-VpnConnectionProxy.md

### PR DESCRIPTION
**Pull request by msarcletti**
Adding additional information for the scope / limitation of the VPN proxy settings configuration

### CODE REVIEW

**Comment by e0i** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by e0i** [comment]
@dariomws @get-itips @tseward

Peer review needed for this PR. Thanks.

**Review by dariomws — APPROVED** [review]
I think the same note should be added to W2012 and W2012 R2 articles.

**Review by get-itips — CHANGES_REQUESTED** [review]
We can not use the [!NOTE] tag on cmdlet's articles, it will not render ok on Get-Help output.
Thanks for the contribution!

File: docset/windows/vpnclient/Set-VpnConnectionProxy.md [hunk 1]
```diff
@@ -35,6 +35,9 @@ Set-VpnConnectionProxy [-AutoDetect] [-AutoConfigurationScript <String>] [-Proxy
 The **Set-VpnConnectionProxy** cmdlet configures web proxy information for the specified VPN connection.
 If errors occur when you modify the web proxy information, the error information is returned.
 
+>[!NOTE]
```
Comment (JohanFreelancer9) [inline docset/windows/vpnclient/Set-VpnConnectionProxy.md:38]:
```suggestion
```

File: docset/windows/vpnclient/Set-VpnConnectionProxy.md [hunk 2]
```diff
@@ -35,6 +35,9 @@ Set-VpnConnectionProxy [-AutoDetect] [-AutoConfigurationScript <String>] [-Proxy
...
+> VPN proxy settings are only used on Force Tunnel Connections. On Split Tunnel Connections, the general proxy settings are used.
```
Comment (JohanFreelancer9) [inline docset/windows/vpnclient/Set-VpnConnectionProxy.md:39]:
```suggestion
Note that VPN proxy settings are only used on forced tunnel connections. On split tunnel connections, the general proxy settings are used.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by tseward — APPROVED** [review]

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by get-itips — APPROVED** [review]

**Pull request merged**